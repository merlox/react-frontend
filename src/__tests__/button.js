import React from 'react'
import { create } from 'react-test-renderer'

function Button (props) {
    return <button>This is a button</button>
}

describe('button component', () => {
    test('matches the screenshot', () => {
        const button = create(<Button />)
        expect(button.toJSON()).toMatchSnapshot()
    })
})
