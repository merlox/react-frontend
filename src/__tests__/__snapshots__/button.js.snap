// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`button component matches the screenshot 1`] = `
<button>
  This is a button
</button>
`;
